packages/openssl/packaging

```diff
@@ -0,0 +1,11 @@
+# abort script on any command that exits with a non zero value
+set -e -x
+
+mkdir ${BOSH_INSTALL_TARGET}/openssl
+echo "Extracting openssl"
+tar xzf openssl/openssl-1.0.2j.tar.gz
```

Comment:
I initiated a brief documentation skeletton in the PR text. 

We need to complete the prereqs for consumming the VIP, in particular check whether bosh stemcells are adding protections against ARP spoofing that would prevent usage of the VIP. See https://www.pivotaltracker.com/n/projects/956238/search?q=arp%20includedone%3Atrue for ARP related manipulations in the bosh agent. So far, I merely saw purges regular of ARP entries, but worth double checking that with bosh specifications (within the bosh repo)